app/app.ts

```diff
@@ -131,7 +117,10 @@ export class ChemiscopeApp {
      */
 
     public async load(configuration: Configuration, dataset: Dataset): Promise<void> {
-        cleanOutdatedMessages();
+        // hide any error coming from the previous dataset loading
+        const errors = getByID('error-display');
+        errors.style.display = 'none';
+
```

Comment:
Will this leave the internals of the element intact? I.e. if you set the display to none and then reset the display, will the innertext be intact?

Reply:
Yes the inner text stays the same, but will be overwritten before the error box is shown again with `style.display = "block"`